electricpy/visu.py

```diff
@@ -321,4 +325,209 @@ def get_output_power(self):
         # low slip
         p_x_2 = center_x - self.radius * _np.cos(beta_1)
         p_y_2 = center_y + self.radius * _np.sin(beta_1)
-        return [p_x_1, p_y_1], [p_x_2, p_y_2]
\ No newline at end of file
+        return [p_x_1, p_y_1], [p_x_2, p_y_2]
+
+class PowerCircle:
+    r"""
+    Plot Power Circle Diagram of Transmission System.
+
+    This class is designed to plot the power circle diagram of a transmission
+    system both sending and reciving ends.
+
+    Parameters
+    ----------
+    type: str "sending" or "receiving"
+
```

Comment:
We don't need an extra space between every input line, they can be removed for considerable savings of visual real-estate.